controllers/fio/job.go

```diff
@@ -62,14 +62,15 @@ func NewJob(cr *perfv1alpha1.Fio, configMap *corev1.ConfigMap, pvcName *string)
 			Name: "custom-jobs", MountPath: "/custom-jobs",
 		})
 	}
-	if pvcName != nil {
+	if cr.Spec.Volume != nil {
+		volumeSource := corev1.VolumeSource{}
```

Comment:
I think we need to add emptyDir as volumeSource. At least in yaml that's what you'd do:
https://kubernetes.io/docs/concepts/storage/volumes/

Reply:
No, I tested it, the above code creates an emptydir.